contracts/ZkDaiBase.sol

```diff
@@ -0,0 +1,73 @@
+pragma solidity ^0.4.25;
+
+contract ZkDaiBase {
```

Comment:
I would make it very clear through comments how the storage inheritance of this system works. I always get a bit uneasy when i see diamond inheritance in solidity

Reply:
Absolutely! Documentation here is definitely required.